Stratis.Bitcoin/Configuration/Settings/StoreSettings.cs

```diff
@@ -14,12 +17,32 @@ public class StoreSettings
         /// <summary><c>true</c> to enable pruning to reduce storage requirements by enabling deleting of old blocks.</summary>
         public bool Prune { get; set; }
 
+        private Action<StoreSettings> callback = null;
+
+        public StoreSettings()
+        {
+        }
+
+        public StoreSettings(Action<StoreSettings> callback)
+            :this()
+        {
+            this.callback = callback;
+        }
+
+        public StoreSettings(NodeSettings nodeSettings, Action<StoreSettings> callback = null)
+            :this(callback)
+        {
+            Load(nodeSettings);
```

Comment:
Please use "this." to mark instance methods.

Reply:
Done